generators/add-web-assets/exc-react/templates/src/index.js

```diff
@@ -49,9 +49,6 @@ function bootstrapInExcShell () {
   // get the Experience Cloud Runtime object
   const runtime = Runtime()
 
-  // set the app name to the Shell header
-  runtime.customEnvLabel = '<%= projectName %>'
```

Comment:
@purplecabbage Is this no longer required?

Reply:
It is ugly. Define 'required' ;)

Reply:
The intended use of this is to bring attention to 'beta' releases, or other environmental factors.
> " ... using it as an app title is not recommended."

Reply:
I thought FF apps were required to show this on EXC shell, 
But now I see, it is to show env and not app name